api/conf/conf.go

```diff
@@ -36,8 +36,8 @@ const (
 	EnvDEV   = "dev"
 	EnvLOCAL = "local"
 
-	confPath   = "/go/manager-api/conf.json"
-	schemaPath = "/go/manager-api/schema.json"
+	confJsonPath = "/go/manager-api/conf.json"
```

Comment:
One PR for one thing, is it related to the PR?

Reply:
fix lint err.